src/fiber/aarch64.cr

```diff
@@ -0,0 +1,65 @@
+{% skip_file unless flag?(:aarch64) %}
+
+class Fiber
+  # :nodoc:
+  def makecontext(stack_ptr, fiber_main) : Void*
+    # align the stack pointer to 16 bytes
+    stack_ptr = Pointer(Void*).new(stack_ptr.address & ~0x0f_u64)
```

Comment:
The alignment was done for every arch before. Wouldn't be better to keep that before the call to `makecontext`?

It would be great to leave somewhere a doc related to `makecontext` and `swapcontext` (and their required annotations).

Reply:
It's still done for every arch but that's probably not required on all architectures, so evolutions can choose to align or not. Is this required for something (like BDWGC)?

Yes, there should be some documentation for the arch-specific methods (that take their names off the deprecated POSIX makecontext / setcontext / swapcontext). I just wasn't exactly sure where to put them. Maybe in `fiber.cr` itself?

Reply:
I am not sure it is a requirement. But it itches me to repeat that "cryptic" call in every method.
Documentation in `fiber.cr` seems good.